src/gui/print_widget.cpp

```diff
@@ -301,6 +301,10 @@ PrintWidget::PrintWidget(Map* map, MainWindow* main_window, MapView* main_view,
 	layout->addRow(world_file_check);
 	world_file_check->hide();
 	
+	transparent_background_check = new QCheckBox(tr("Make background transparent"));
```

Comment:
We cannot enforce transparent background, but we can enforce white background.

Looking for feedback: Should this be 

 - [x] Draw white background

For formats without transparency, drawing the white background would be done implicitly.
We could completely avoid the error message. 
(But we might see questions why the background is always white in JPEG format.)

Reply:
That change is indeed an option as making the background transparent is sometimes a unrealizable promise.
OTOH a user might wonder what happens if he deselects the checkmark: will the background be black (BTW: this happens if Mapper would allow transparency for .bmp) as he might not think about transparency?
So I personally have a preference for the current wording but I'm fine with the alternative.